docs/reference/glossary.rst

```diff
@@ -119,6 +119,14 @@ Glossary
     Replication Factor (RF)
       The total number of replica nodes across a given cluster. An :abbr:`RF (Replication Factor)` of 1 means that the data will only exist on a single node in the cluster and will not have any fault tolerance. This number is a setting defined for each keyspace. All replicas share equal priority; there are no primary or master replicas. An RF for any table, can be defined for each :abbr:`DC (Data Center)`. See :doc:`Fault Tolerance </architecture/architecture-fault-tolerance>`.
 
+    Repair Based Node Operations (RBNO)
+      :abbr:`RBNO (Repair Based Node Operations)` is an internal ScyllaDB mechanism that uses repair to 
+      synchronize data between the nodes in a cluster instead of using streaming. RBNO significantly 
```

Comment:
IMHO RBNO and repair also use streaming
But I will let @asias to comment

Reply:
@tzach I took the info from a presentation @asias gave at the summit. But yes, let's wait, especially since I want to update the other page about RBNO.

Reply:
RBNO which uses repair under the hood. RBNO and repair do not use streaming since row level repair was introduced.